conans/client/client_cache.py

```diff
@@ -210,6 +210,7 @@ def conan_builds(self, conan_reference):
 
     def load_manifest(self, conan_reference):
```

Comment:
Asserts in this file only added because there are present in other methods, to be consistent, not really related with the revisions feature.